.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Please ignore the `.idea` folder and remove all those files from git.
Ex: https://github.com/leanovate/play-mockws/blob/master/.gitignore#L2